devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -864,30 +897,43 @@ bool WholeBodyDynamicsDevice::openMultipleAnalogSensorRemapper(os::Searchable &c
     yarp::os::Property prop;
     prop.fromString(config.toString().c_str());
     yarp::os::Property propMASRemapper;
-    yarp::os::Bottle & propMasNames=prop.findGroup("multipleAnalogSensorsNames");
+
     propMASRemapper.put("device","multipleanalogsensorsremapper");
-    bool ok=false;
-    for (auto types=1u;types<propMasNames.size();types++){
-        yarp::os::Bottle * mas_type = propMasNames.get(types).asList();
-        if( mas_type->size() != 2 || mas_type->get(1).asList() == nullptr )
-        {
-            yError() << "wholeBodyDynamics: multipleAnalogSensorsNames group is malformed (" << mas_type->toString() << "). ";
-            return false;
-        }
-        else {
-            ok=true;
-        }
 
-        propMASRemapper.addGroup(mas_type->get(0).asString());
-        yarp::os::Bottle  MASnames;
-        yarp::os::Bottle & MASnamesList= MASnames.addList();
-        for(auto i=0u; i < mas_type->get(1).asList()->size(); i++)
-        {
-            MASnamesList.addString(mas_type->get(1).asList()->get(i).asString());
-        }
-        propMASRemapper.put(mas_type->get(0).asString(),MASnames.get(0));
+    // We get the multipleAnalogSensorsNames group, but from it we just use the multipleAnalogSensorsNames parameter
+    if (!prop.check("multipleAnalogSensorsNames"))
+    {
+        yError() << "wholeBodyDynamics :Impossible to find requied group multipleAnalogSensorsNames";
+        return false;
+    }
+
+    yarp::os::Searchable& multipleAnalogSensorsNames = prop.findGroup("multipleAnalogSensorsNames");
+
+    bool required = true;
+    bool ok = getConfigParamsAsList(multipleAnalogSensorsNames, "SixAxisForceTorqueSensorsNames", this->ftSensorNames, required);
+    if (!ok)
+    {
+        yError() << "wholeBodyDynamics :Impossible to find required variable SixAxisForceTorqueSensorsNames in group multipleAnalogSensorsNames";
+        return false;
+    }
 
+    if (multipleAnalogSensorsNames.find("TemperatureSensorsNames").isList())
+    {
+        yWarning() << "wholeBodyDynamics : TemperatureSensorsNames parameter is not used anymore, the names in SixAxisForceTorqueSensorsNames are used, please remove TemperatureSensorsNames parameter to avoid confusion";
```

Comment:
```suggestion
        yWarning() << "wholeBodyDynamics: TemperatureSensorsNames parameter is not used anymore, the names in SixAxisForceTorqueSensorsNames are used, please remove TemperatureSensorsNames parameter to avoid confusion";
```